src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -93,7 +93,7 @@ protected function configure(): void
 
     <info>php %command.full_name% <receiver-name> --memory-limit=128M</info>
 
-Use the --time-limit option to stop the worker when the given time limit (in seconds) is reached:
```

Comment:
I'm not sure this is more clear: the sentence in parentheses introduced confusion te me, because of the negation. At least the previous sentence was not confusing (it was not completely accurate).

Reply:
@nicolas-grekas I've rephrased it: does it seem better to you?

Reply:
`Use the --time-limit option to stop the worker when the given time limit (in seconds) is reached (if a message is being handled, the worker will stop after the processing finished):`

Reply:
@nicolas-grekas thank you for your proposal, I've updated the code accordingly